cypress/support/e2e.js

```diff
@@ -6,6 +6,14 @@ import chaiExtension from './chai.js'
 
 before(() => {
 	chai.use(chaiExtension)
+
+	Cypress.on('uncaught:exception', (err) => {
+		if (err.message.includes('ResizeObserver')) {
+		  return false
+		}
+
+		return true
+	})
```

Comment:
I ran into this one as well when working on 28 cypress tests. Any idea where it's coming from?

Reply:
Could be either text itself for the menubar since #4637 or apps/files/src/components/VirtualList.vue for example, but as far as I read this is safe to just ignore.

Reply:
https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded